src/entity/UserAction.ts

```diff
@@ -1,7 +1,13 @@
 import { Column, Entity, Index, PrimaryColumn } from 'typeorm';
 
 export enum UserActionType {
-  Notification = 'notification',
+  Notification = 'enabled_notification',
```

Comment:
Let's keep the same convention

```suggestion
  Notification = 'enable_notification',
```